### ISSUE TITLE
If Test VI is closed, double-click on test in test results will cause an error

### ISSUE THREAD

**Issue by jimkring** [JKISoftware/Caraya#28]
- when test VI, which ran tests is closed, and user double-clicks on test result in the results list, error occurs (b/c VI is not in the memory anymore).

![double-click while test vi is closed](https://user-images.githubusercontent.com/1911151/28896941-df025980-77de-11e7-92aa-58b639f8852a.png)

See pull request https://github.com/JKISoftware/Caraya/pull/12 - Was fixed in a way, that by Application Dir, search for VI by name, and if there is just 1 VI with this name, then open its BD by path, not by name.

**Comment by kosist** [comment]
Tested with release candidate 0.6.0.
Actually, on existing test library, I had still this behaviour, but just for tests launcher.
When new project was created, it was not observed.
But as soon as I've moved some test VIs into library, then error was present.
I attach test project (LV 2017 SP1).

Run test via Test Launcher.vi -> by double-click it is possible to navigate via test nodes, and test VIs.
When close Test Launcher.vi (and other VIs) -> error is dropped for vis, which are inside the library.
![screenshot 2018-10-24 14 56 58](https://user-images.githubusercontent.com/1911151/47433065-e960b080-d79f-11e8-9c15-cb68e24d6616.png)

If VI stays opened, then no error is dropped.
For the VI, which is not in the library, no error is dropped, even if all VIs are closed.
[TestCaraya.zip](https://github.com/JKISoftware/Caraya/files/2510623/TestCaraya.zip)
